### ISSUE TITLE
Compiler crashes in JTReg tests

### ISSUE THREAD

**Issue by adam-thorpe** [eclipse/openj9#8988]
Seeing lots of compiler crashes in various JTReg tests this morning, across multiple platforms and versions.
https://ci.adoptopenjdk.net/user/adam-thorpe/my-views/view/OpenJ9%20sanity%20openjdk/job/Test_openjdk11_j9_sanity.openjdk_x86-64_windows/160/consoleFull

FIRSTBAD
02:29:54  OpenJ9   - 752c590a2
02:29:54  OMR      - e9b85117d
02:29:54  JCL      - d1ca0ebebe based on jdk-11.0.7+8)

LASTGOOD
00:45:28  OpenJ9   - 398fa6730
00:45:28  OMR      - dc5747bc9
00:45:28  JCL      - d1ca0ebebe based on jdk-11.0.7+8)

https://github.com/eclipse/openj9/compare/398fa6730...752c590a2
https://github.com/eclipse/openj9-omr/compare/dc5747bc9...e9b85117d

eg.
```
02:41:10  An exception has occurred in the compiler (11.0.7). Please file a bug against the Java compiler via the Java bug reporting page (http://bugreport.java.com) after checking the Bug Database (http://bugs.java.com) for duplicates. Include your program and the following diagnostic in your report. Thank you.
02:41:10  java.lang.NullPointerException
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.flags(Symbol.java:1259)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Check.checkSunAPI(Check.java:3278)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.checkIdInternal(Attr.java:3933)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.checkId(Attr.java:3804)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.visitSelect(Attr.java:3696)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCFieldAccess.accept(JCTree.java:2114)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribTree(Attr.java:655)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:715)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:708)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.attribImportType(TypeEnter.java:441)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.doImport(TypeEnter.java:424)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.resolveImports(TypeEnter.java:364)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.runPhase(TypeEnter.java:321)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.doCompleteEnvs(TypeEnter.java:282)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:251)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter.complete(TypeEnter.java:198)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Enter.complete(Enter.java:583)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.readSourceFile(JavaCompiler.java:841)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.readSourceFile(JavaCompiler.java:797)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.lambda$new$0(JavaCompiler.java:356)
02:41:10  	at com.sun.tools.javac.main.JavaCompiler$$Lambda$40/0000000000000000.complete(Unknown Source)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.fillIn(ClassFinder.java:363)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.complete(ClassFinder.java:291)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder$$Lambda$41/0000000000000000.complete(Unknown Source)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.loadClass(ClassFinder.java:433)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.loadClass(Resolve.java:1996)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.findIdentInPackageInternal(Resolve.java:2386)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.findIdentInPackage(Resolve.java:2374)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.selectSym(Attr.java:3716)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.visitSelect(Attr.java:3601)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCFieldAccess.accept(JCTree.java:2114)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribTree(Attr.java:655)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribImportQualifier(Attr.java:390)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.doImport(TypeEnter.java:410)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.resolveImports(TypeEnter.java:364)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$ImportsPhase.runPhase(TypeEnter.java:321)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.doCompleteEnvs(TypeEnter.java:282)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:251)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter.complete(TypeEnter.java:198)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Enter.complete(Enter.java:583)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.readSourceFile(JavaCompiler.java:841)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.readSourceFile(JavaCompiler.java:797)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.lambda$new$0(JavaCompiler.java:356)
02:41:10  	at com.sun.tools.javac.main.JavaCompiler$$Lambda$40/0000000000000000.complete(Unknown Source)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.fillIn(ClassFinder.java:363)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.complete(ClassFinder.java:291)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder$$Lambda$41/0000000000000000.complete(Unknown Source)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.ClassFinder.loadClass(ClassFinder.java:433)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.loadClass(Resolve.java:1996)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.findGlobalType(Resolve.java:2243)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.findType(Resolve.java:2322)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.findIdentInternal(Resolve.java:2355)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.findIdent(Resolve.java:2341)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Resolve.resolveIdent(Resolve.java:2602)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.visitIdent(Attr.java:3488)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCIdent.accept(JCTree.java:2248)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribTree(Attr.java:655)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:715)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:708)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.visitTypeApply(Attr.java:4213)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCTypeApply.accept(JCTree.java:2380)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribTree(Attr.java:655)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:715)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:708)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.visitWildcard(Attr.java:4399)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCWildcard.accept(JCTree.java:2508)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribTree(Attr.java:655)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:715)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:708)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribAnyTypes(Attr.java:766)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribTypes(Attr.java:774)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.visitTypeApply(Attr.java:4216)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCTypeApply.accept(JCTree.java:2380)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribTree(Attr.java:655)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:715)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Attr.attribType(Attr.java:708)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.MemberEnter.visitVarDef(MemberEnter.java:264)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCVariableDecl.accept(JCTree.java:956)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.MemberEnter.memberEnter(MemberEnter.java:163)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.MemberEnter.signature(MemberEnter.java:111)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.MemberEnter.visitMethodDef(MemberEnter.java:193)
02:41:10  	at jdk.compiler/com.sun.tools.javac.tree.JCTree$JCMethodDecl.accept(JCTree.java:866)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.MemberEnter.memberEnter(MemberEnter.java:163)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.MemberEnter.memberEnter(MemberEnter.java:175)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$MembersPhase.finishClass(TypeEnter.java:958)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$MembersPhase.runPhase(TypeEnter.java:942)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.doCompleteEnvs(TypeEnter.java:282)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$MembersPhase.doCompleteEnvs(TypeEnter.java:877)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:251)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:266)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:266)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter$Phase.completeEnvs(TypeEnter.java:266)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.TypeEnter.complete(TypeEnter.java:198)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol.complete(Symbol.java:642)
02:41:10  	at jdk.compiler/com.sun.tools.javac.code.Symbol$ClassSymbol.complete(Symbol.java:1326)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Enter.complete(Enter.java:583)
02:41:10  	at jdk.compiler/com.sun.tools.javac.comp.Enter.main(Enter.java:560)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.enterTrees(JavaCompiler.java:1066)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:937)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:311)
02:41:10  	at jdk.compiler/com.sun.tools.javac.main.Main.compile(Main.java:170)
02:41:10  	at jdk.compiler/com.sun.tools.javac.Main.compile(Main.java:75)
02:41:10  	at jdk.internal.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
02:41:10  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
02:41:10  	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
02:41:10  	at com.sun.javatest.regtest.agent.RegressionCompileCommand.run(RegressionCompileCommand.java:189)
02:41:10  	at com.sun.javatest.regtest.agent.CompileActionHelper.runCompile(CompileActionHelper.java:92)
02:41:10  	at com.sun.javatest.regtest.agent.AgentServer.doCompile(AgentServer.java:220)
02:41:10  	at com.sun.javatest.regtest.agent.AgentServer.run(AgentServer.java:190)
02:41:10  	at com.sun.javatest.regtest.agent.AgentServer.main(AgentServer.java:65)
02:41:10  
02:41:10  TEST RESULT: Error. compiler crashed (exit code 4)`
```

example stack trace from xlinux run
```
04:29:37  STDERR:
04:29:37  Unhandled exception
04:29:37  Type=Segmentation error vmState=0x00000000
04:29:37  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
04:29:37  Handler1=00007F39EA233EE0 Handler2=00007F39E9B259F0 InaccessibleAddress=00007F4959D5F7F0
04:29:37  RDI=FFFFFFF8038A2790 RSI=00007F415AEB4D40 RAX=FFFFFFF8038A2790 RBX=0000000000000000
04:29:37  RCX=FFFFFFF8038A2790 RDX=00000007FEEAAAC0 R8=0000000002711CE0 R9=00007F415AEB4D40
04:29:37  R10=000000000251DD08 R11=00007F39916B44E6 R12=0000000000000008 R13=000000000274D248
04:29:37  R14=00007F395C009AC8 R15=00007F39918E5EB8
04:29:37  RIP=00007F39EB7E73D0 GS=0000 FS=0000 RSP=00007F3990042CB8
04:29:37  EFlags=0000000000010216 CS=0033 RBP=0000000002741A00 ERR=0000000000000004
04:29:37  TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007F4959D5F7F0
04:29:37  xmm0 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm2 00000000ffe13e18 (f: 4292951552.000000, d: 2.121000e-314)
04:29:37  xmm3 00000000ffdd5648 (f: 4292695552.000000, d: 2.120873e-314)
04:29:37  xmm4 0000002900001958 (f: 6488.000000, d: 8.700183e-313)
04:29:37  xmm5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm6 0000000002741a00 (f: 41163264.000000, d: 2.033735e-316)
04:29:37  xmm7 00000000e01bda18 (f: 3759921664.000000, d: 1.857648e-314)
04:29:37  xmm8 4081880000000000 (f: 0.000000, d: 5.610000e+02)
04:29:37  xmm9 0000000040000000 (f: 1073741824.000000, d: 5.304989e-315)
04:29:37  xmm10 000000003f800000 (f: 1065353216.000000, d: 5.263544e-315)
04:29:37  xmm11 4130080a5046bc2f (f: 1346812928.000000, d: 1.050634e+06)
04:29:37  xmm12 000000004158e095 (f: 1096343680.000000, d: 5.416658e-315)
04:29:37  xmm13 0000000048bed09b (f: 1220464768.000000, d: 6.029897e-315)
04:29:37  xmm14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm15 40c922779ebc6f31 (f: 2663149312.000000, d: 1.286893e+04)
04:29:37  Module=/lib/x86_64-linux-gnu/libc.so.6
04:29:37  Module_base_address=00007F39EB691000
04:29:37  Target=2_90_20200326_547 (Linux 4.4.0-159-generic)
04:29:37  CPU=amd64 (4 logical CPUs) (0x1f225e000 RAM)
04:29:37  ----------- Stack Backtrace -----------
04:29:37  (0x00007F39EB7E73D0 [libc.so.6+0x1563d0])
04:29:37  (0x00007F39E3B4AE29 [libj9jit29.so+0x920e29])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4B151 [libj9jit29.so+0x921151])
04:29:37  (0x00007F39E3B4C735 [libj9jit29.so+0x922735])
04:29:37  (0x00007F39E3B5AE1C [libj9jit29.so+0x930e1c])
04:29:37  ---------------------------------------
04:29:37  JVMDUMP039I Processing dump event "gpf", detail "" at 2020/03/26 04:29:15 - please wait.
04:29:37  Unhandled exception
04:29:37  Type=Segmentation error vmState=0x00000000
04:29:37  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
04:29:37  Handler1=00007F39EA233EE0 Handler2=00007F39E9B259F0 InaccessibleAddress=00007F495E071AB0
04:29:37  RDI=FFFFFFF803722040 RSI=00007F415F03F800 RAX=FFFFFFF803722040 RBX=0000000000000000
04:29:37  RCX=FFFFFFF803722040 RDX=00000007FF0322C0 R8=0000000002711CE0 R9=00007F415F03F800
04:29:37  R10=000000000251DD08 R11=00007F39916B44E6 R12=0000000000000008 R13=00000000027542F8
04:29:37  R14=00007F396000D448 R15=00007F39918E5EB8
04:29:37  RIP=00007F39EB7E73D0 GS=0000 FS=0000 RSP=00007F3990083CB8
04:29:37  EFlags=0000000000010216 CS=0033 RBP=000000000273DE00 ERR=0000000000000004
04:29:37  TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007F495E071AB0
04:29:37  xmm0 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm2 00000000ffe13e18 (f: 4292951552.000000, d: 2.121000e-314)
04:29:37  xmm3 00000000ffe06470 (f: 4292895744.000000, d: 2.120972e-314)
04:29:37  xmm4 000000290000191d (f: 6429.000000, d: 8.700183e-313)
04:29:37  xmm5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm6 000000000273de00 (f: 41147904.000000, d: 2.032977e-316)
04:29:37  xmm7 00000000e01bda18 (f: 3759921664.000000, d: 1.857648e-314)
04:29:37  xmm8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm10 000000003f800000 (f: 1065353216.000000, d: 5.263544e-315)
04:29:37  xmm11 412ff6f4400bcc8e (f: 1074515072.000000, d: 1.047418e+06)
04:29:37  xmm12 00000000415aa98c (f: 1096460672.000000, d: 5.417236e-315)
04:29:37  xmm13 0000000048c062a9 (f: 1220567680.000000, d: 6.030406e-315)
04:29:37  xmm14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm15 40c85e22e4bee434 (f: 3837715456.000000, d: 1.247627e+04)
04:29:37  Unhandled exception
04:29:37  Type=Segmentation error vmState=0x00000000
04:29:37  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
04:29:37  Handler1=00007F39EA233EE0 Handler2=00007F39E9B259F0 InaccessibleAddress=00007F4955CC4B30
04:29:37  RDI=FFFFFFF803729640 RSI=00007F4156C92D80 RAX=FFFFFFF803729640 RBX=0000000000000000
04:29:37  RCX=FFFFFFF803729640 RDX=00000007FF031DC0 R8=0000000002711CE0 R9=00007F4156C92D80
04:29:37  R10=000000000251DD08 R11=00007F39916B44E6 R12=0000000000000008 R13=000000000275B3F8
04:29:37  R14=00007F39580033C8 R15=00007F39918E5EB8
04:29:37  RIP=00007F39EB7E73D0 GS=0000 FS=0000 RSP=00007F396B595CB8
04:29:37  EFlags=0000000000010216 CS=0033 RBP=0000000002745600 ERR=0000000000000004
04:29:37  TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007F4955CC4B30
04:29:37  xmm0 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm2 00000000ffe13e18 (f: 4292951552.000000, d: 2.121000e-314)
04:29:37  xmm3 00000000ffd91f30 (f: 4292419328.000000, d: 2.120737e-314)
04:29:37  xmm4 00000029000015d0 (f: 5584.000000, d: 8.700183e-313)
04:29:37  xmm5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm6 0000000002745600 (f: 41178624.000000, d: 2.034494e-316)
04:29:37  xmm7 00000000e01bda18 (f: 3759921664.000000, d: 1.857648e-314)
04:29:37  xmm8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm10 000000003e8c05e2 (f: 1049363968.000000, d: 5.184547e-315)
04:29:37  xmm11 000000003c075284 (f: 1007112832.000000, d: 4.975799e-315)
04:29:37  xmm12 0000000040400000 (f: 1077936128.000000, d: 5.325712e-315)
04:29:37  xmm13 0000000048bed09b (f: 1220464768.000000, d: 6.029897e-315)
04:29:37  xmm14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm15 3ff0000000000000 (f: 0.000000, d: 1.000000e+00)
04:29:37  Module=/lib/x86_64-linux-gnu/libc.so.6
04:29:37  Module_base_address=00007F39EB691000
04:29:37  Target=2_90_20200326_547 (Linux 4.4.0-159-generic)
04:29:37  CPU=amd64 (4 logical CPUs) (0x1f225e000 RAM)
04:29:37  ----------- Stack Backtrace -----------
04:29:37  (0x00007F39EB7E73D0 [libc.so.6+0x1563d0])
04:29:37  (0x00007F39E3B4AE29 [libj9jit29.so+0x920e29])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4B151 [libj9jit29.so+0x921151])
04:29:37  (0x00007F39E3B4C735 [libj9jit29.so+0x922735])
04:29:37  (0x00007F39E3B5AE1C [libj9jit29.so+0x930e1c])
04:29:37  ---------------------------------------
04:29:37  JVMDUMP039I Processing dump event "gpf", detail "" at 2020/03/26 04:29:15 - please wait.
04:29:37  Module=/lib/x86_64-linux-gnu/libc.so.6
04:29:37  Module_base_address=00007F39EB691000
04:29:37  Target=2_90_20200326_547 (Linux 4.4.0-159-generic)
04:29:37  CPU=amd64 (4 logical CPUs) (0x1f225e000 RAM)
04:29:37  ----------- Stack Backtrace -----------
04:29:37  (0x00007F39EB7E73D0 [libc.so.6+0x1563d0])
04:29:37  (0x00007F39E3B4AE29 [libj9jit29.so+0x920e29])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4B151 [libj9jit29.so+0x921151])
04:29:37  (0x00007F39E3B4C735 [libj9jit29.so+0x922735])
04:29:37  (0x00007F39E3B5AE1C [libj9jit29.so+0x930e1c])
04:29:37  ---------------------------------------
04:29:37  JVMDUMP039I Processing dump event "gpf", detail "" at 2020/03/26 04:29:15 - please wait.
04:29:37  Unhandled exception
04:29:37  Type=Segmentation error vmState=0x00000000
04:29:37  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
04:29:37  Handler1=00007F39EA233EE0 Handler2=00007F39E9B259F0 InaccessibleAddress=00007F496E1E9E00
04:29:37  RDI=FFFFFFF80356F4A0 RSI=00007F416F148D50 RAX=FFFFFFF80356F4A0 RBX=0000000000000000
04:29:37  RCX=FFFFFFF80356F4A0 RDX=00000007FF0A10C0 R8=0000000002711CE0 R9=00007F416F148D50
04:29:37  R10=000000000251DD08 R11=00007F39916B44E6 R12=0000000000000008 R13=0000000002610558
04:29:37  R14=00007F39700A7B98 R15=00007F39918E5EB8
04:29:37  RIP=00007F39EB7E73D0 GS=0000 FS=0000 RSP=00007F39903097D8
04:29:37  EFlags=0000000000010216 CS=0033 RBP=00000000025C0C00 ERR=0000000000000004
04:29:37  TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007F496E1E9E00
04:29:37  xmm0 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm2 00000000ffe13e18 (f: 4292951552.000000, d: 2.121000e-314)
04:29:37  xmm3 00000000ffe14230 (f: 4292952576.000000, d: 2.121000e-314)
04:29:37  xmm4 00000000ffe13dc8 (f: 4292951552.000000, d: 2.121000e-314)
04:29:37  xmm5 00000000ffe13dc8 (f: 4292951552.000000, d: 2.121000e-314)
04:29:37  xmm6 0000000002717f00 (f: 40992512.000000, d: 2.025299e-316)
04:29:37  xmm7 00000000ffe0f960 (f: 4292933888.000000, d: 2.120991e-314)
04:29:37  xmm8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm10 000000003e8c05e2 (f: 1049363968.000000, d: 5.184547e-315)
04:29:37  xmm11 000000003c075284 (f: 1007112832.000000, d: 4.975799e-315)
04:29:37  xmm12 0000000040400000 (f: 1077936128.000000, d: 5.325712e-315)
04:29:37  xmm13 0000000048bed09b (f: 1220464768.000000, d: 6.029897e-315)
04:29:37  xmm14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
04:29:37  xmm15 40c89ea1e9d6b2d1 (f: 3923161856.000000, d: 1.260526e+04)
04:29:37  Module=/lib/x86_64-linux-gnu/libc.so.6
04:29:37  Module_base_address=00007F39EB691000
04:29:37  Target=2_90_20200326_547 (Linux 4.4.0-159-generic)
04:29:37  CPU=amd64 (4 logical CPUs) (0x1f225e000 RAM)
04:29:37  ----------- Stack Backtrace -----------
04:29:37  (0x00007F39EB7E73D0 [libc.so.6+0x1563d0])
04:29:37  (0x00007F39E3B4AE29 [libj9jit29.so+0x920e29])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4ADBF [libj9jit29.so+0x920dbf])
04:29:37  (0x00007F39E3B4B151 [libj9jit29.so+0x921151])
04:29:37  (0x00007F39E3B4C735 [libj9jit29.so+0x922735])
04:29:37  (0x00007F39E3B5AE1C [libj9jit29.so+0x930e1c])
```

**Comment by DanHeidinga** [comment]
Tagging as jit on the basis of:
```
04:29:37  ----------- Stack Backtrace -----------
04:29:37  (0x00007F39EB7E73D0 [libc.so.6+0x1563d0])
04:29:37  (0x00007F39E3B4AE29 [libj9jit29.so+0x920e29])
```

fyi @andrewcraik

**Comment by andrewcraik** [comment]
@liqunl could you take a look please?

**Comment by pshipton** [comment]
Seems to be resolved by #8995
